.gitignore

```diff
@@ -56,6 +56,7 @@ coverage.xml
 *.py,cover
 .hypothesis/
 .pytest_cache/
+python/adelphi/tox.ini
```

Comment:
I'm wondering if we want to be even more aggressive and not include tox.ini anywhere in the repo... although as a practical matter what you have here will cover the overwhelmingly common case.  I'm fine with this.